emscripten.py

```diff
@@ -2213,8 +2251,11 @@ def debug_copy(src, dst):
     shared.check_call(sourcemap_cmd)
     debug_copy(base_source_map, 'base_wasm.map')
 
-  cmd = [wasm_emscripten_finalize, base_wasm, '-o', wasm,
-         '--global-base=%s' % shared.Settings.GLOBAL_BASE]
+  cmd = [wasm_emscripten_finalize, base_wasm, '-o', wasm]
+  if shared.Settings.RELOCATABLE:
+    cmd.append('--global-base=0')
```

Comment:
is this just ignored for relocatable modules? or is the value 0 somehow important?

Reply:
Added comment.